debian/clean

```diff
@@ -0,0 +1,2 @@
+include/libtpms/*.gch
+tests/NVChip
```

Comment:
This signals `debian/rules clean` to remove these files (which are otherwise not automatically picked up) as part of the `clean` target.